### DISCUSSION TITLE
Support for .Net 6 Winforms

### DISCUSSION THREAD

**Discussion by SuperDaveOsbourne** [Live-Charts/LiveCharts2#302] [General]
Is there support for .Net 6 Winforms?

**Comment by beto-rodriguez** [comment]
Yes, LiveCharts2 works in net 6

**Comment by SuperDaveOsbourne** [comment]
Is there a way to put it in winforms? I tried to add it to the toolbox, no joy for the beta version.

**Comment by beto-rodriguez** [comment]
I just tested it a few days ago and I had no issue with that, I actually built this article:

https://github.com/beto-rodriguez/LiveCharts2/blob/master/docs/overview/1.2.install.md#install-from-nuget-2

The article is a template that the web site will use, so it is not super clear, but the idea is there, maybe it helps you

**Comment by SuperDaveOsbourne** [comment]
Cool, I am using VS 2021 and I had to exit and restart before they showed up.
